api/src/main/resources/metadata/concepts_and_drugs/Concept_Name.xml

```diff
@@ -6797,6 +6797,6 @@
     <concept_name  concept_name_id="107467"  concept_id="159928"  name="school"  locale="en"  locale_preferred="0"  creator="1"  date_created="2010-11-29 01:42:21"  concept_name_type="FULLY_SPECIFIED"  voided="1"  voided_by="1"  date_voided="2024-05-28 12:30:19"  void_reason="Deleted due to change of name"  uuid="107467BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"  date_changed="2024-05-28 12:30:19"  changed_by="1"/>
     <concept_name  concept_name_id="187872"  concept_id="159928"  name="School, other institution, public administrative area"  locale="en"  locale_preferred="1"  creator="1"  date_created="2024-05-28 12:30:19"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="b87bf7b2-77cd-489c-b9df-dd22428b1984"  date_changed="2024-05-28 12:30:19"  changed_by="1"/>
     <concept_name  concept_name_id="198843"  concept_id="186350"  name="arv split drug set"  locale="en"  locale_preferred="1"  creator="1"  date_created="2024-06-18 12:40:41"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="b824100c-2d84-11ef-b0cb-e5066d1008f8"/>
-    <concept_name  concept_name_id="198844"  concept_id="175386"  name="Image attachment"  locale="en"  locale_preferred="1"  creator="2"  date_created="2024-04-14 15:40:17"  concept_name_type="FULLY_SPECIFIED"  voided="0"  uuid="552e757e-b072-4e68-a10d-b7db7efcd5ad" />
-    <concept_name  concept_name_id="198845"  concept_id="175385"  name="Default attachment"  locale="en"  locale_preferred="1"  creator="2"  date_created="2024-04-14 15:40:17"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="209ebadb-d3df-489a-bd10-7171e21c4869" />
+    <concept_name  concept_name_id="198844"  concept_id="186351"  name="Image attachment"  locale="en"  locale_preferred="1"  creator="2"  date_created="2024-04-14 15:40:17"  concept_name_type="FULLY_SPECIFIED"  voided="0"  uuid="552e757e-b072-4e68-a10d-b7db7efcd5ad" />
+    <concept_name  concept_name_id="198845"  concept_id="186352"  name="Default attachment"  locale="en"  locale_preferred="1"  creator="2"  date_created="2024-04-14 15:40:17"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="209ebadb-d3df-489a-bd10-7171e21c4869" />
```

Comment:
@slubwama You also have ATT IMAGE ATTACHMENT and ATT DEFAULT ATTACHMENT. 
Then here you have Image attachment and Default attachment. 
Do you need to have all of them or it was some kind of repetition?

Reply:
When you follow  the concepts being create back to the attachment  module  you will notice  that. They only  get created  when they don't exist. What we are trying  to solve  here is the random  creation  of these concepts by creating  them first before the attachment  module  does.

Reply:
This is noted @slubwama